src/layouts/login/LoginLayout.astro

```diff
@@ -0,0 +1,122 @@
+---
+import BaseLayout from "../BaseLayout.astro";
+import type {
```

Comment:
Most of these imports seem to be unused. Specifically all the imports from `Types`, and the `LoginMethodSelector`. Also, Cookies aren't accessed, so this line can also be deleted.